### PR TITLE
Release 99 Intl.Locale Chromium adds support for some new properties

### LINKED ISSUE #15174: Issue with "Intl.Locale": Browser compatibility Tables (Chrome)
### **Chrome version 99 onwards now fully supports the remaining Instance properties of the **Intl.Locale()** constructor; namely:**

[Intl.Locale.prototype.calendar](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Intl/Locale/calendar)

collations (broken link on page)

[Intl.Locale.prototype.hourCycles](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Intl/Locale/hourCycles)

[Intl.Locale.prototype.numberingSystems](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Intl/Locale/numberingSystems)

[Intl.Locale.prototype.textInfo](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Intl/Locale/textInfo)

[Intl.Locale.prototype.timeZones](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Intl/Locale/timeZones)

[Intl.Locale.prototype.weekInfo](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Intl/Locale/weekInfo)

**The Browser compatibility Tables on the various pages and the main page need to be updated.**

### PR DESCRIPTION
**Pull request by hamishwillee**
From https://v8.dev/blog/v8-release-99#javascript

> [Intl.Locale API](https://v8.dev/blog/v8-release-74#intl.locale). With v9.9, we added seven new properties to the Intl.Locale object: calendars, collations, hourCycles, numberingSystems, timeZones, textInfo, and weekInfo.

This adds the versions. Previously these were behind flags. Note however that the flag info hadn't been extended to Opera, so I have added that too.

Fixes #15174

### CODE REVIEW

**Review by queengooborg — CHANGES_REQUESTED** [review]
Thanks Hamish!  This is mostly LGTM, but Opera Android actually doesn't support flags.  Can we remove flagged data from Opera Android?

File: javascript/builtins/intl/Locale.json [hunk 1]
```diff
@@ -265,11 +290,31 @@
...
+                        "type": "preference",
+                        "name": "#enable-javascript-harmony",
+                        "value_to_set": "Enabled"
+                      }
+                    ]
+                  }
+                ],
                 "opera_android": {
-                  "version_added": false
+                  "version_added": "65",
```
Comment (hamishwillee) [inline javascript/builtins/intl/Locale.json:310]:
```suggestion
                  "version_added": false
```

File: javascript/builtins/intl/Locale.json [hunk 2]
```diff
@@ -265,11 +290,31 @@
...
+                  "flags": [
+                    {
+                      "type": "preference",
+                      "name": "#enable-javascript-harmony",
+                      "value_to_set": "Enabled"
+                    }
+                  ]
```
Comment (hamishwillee) [inline javascript/builtins/intl/Locale.json:317]:
```suggestion
                  "version_added": false
```

File: javascript/builtins/intl/Locale.json [hunk 3]
```diff
@@ -444,11 +502,31 @@
...
```
Comment (hamishwillee) [inline javascript/builtins/intl/Locale.json:529]:
```suggestion
                  "version_added": "65"
```

File: javascript/builtins/intl/Locale.json [hunk 4]
```diff
@@ -568,11 +671,31 @@
...
```
Comment (hamishwillee) [inline javascript/builtins/intl/Locale.json:698]:
```suggestion
                  "version_added": false
```

File: javascript/builtins/intl/Locale.json [hunk 5]
```diff
@@ -857,11 +1005,31 @@
...
```
Comment (hamishwillee) [inline javascript/builtins/intl/Locale.json:1032]:
```suggestion
                  "version_added": false
```

File: javascript/builtins/intl/Locale.json [hunk 6]
```diff
@@ -1091,11 +1284,31 @@
...
```
Comment (hamishwillee) [inline javascript/builtins/intl/Locale.json:1311]:
```suggestion
                  "version_added": false
```

File: javascript/builtins/intl/Locale.json [hunk 7]
```diff
@@ -1160,11 +1398,31 @@
...
```
Comment (hamishwillee) [inline javascript/builtins/intl/Locale.json:1425]:
```suggestion
                  "version_added": false
```

File: javascript/builtins/intl/Locale.json [hunk 8]
```diff
@@ -1284,11 +1567,31 @@
...
```
Comment (hamishwillee) [inline javascript/builtins/intl/Locale.json:1594]:
```suggestion
                  "version_added": false
```

**Comment by hamishwillee** [comment]
Thanks @queengooborg - Fixed!

**Review by queengooborg — APPROVED** [review]
Thank you, LGTM!

**Pull request merged**